docs/source/ref/contrib/gis.rst

```diff
@@ -0,0 +1,26 @@
+=========
+GeoDjango
+=========
+
+.. versionadded:: 5.2.0b2
+
+Django MongoDB Backend supports :doc:`GeoDjango<django:ref/contrib/gis/index>`.
+
+Configuration
+=============
+
+#. Install the necessary :doc:`Geospatial libraries
+   <django:ref/contrib/gis/install/geolibs>` (GEOS and GDAL).
+#. Add :mod:`django.contrib.gis` to :setting:`INSTALLED_APPS` in your settings.
+   This is so that the ``gis`` templates can be located -- if not done, then
+   features such as the geographic admin or KML sitemaps will not function properly.
```

Comment:
Any reservation to having it be an extension for us `django-mongodb-backend[geos]` or something of that nature?